testing/btest/Baseline/scripts.policy.frameworks.netcontrol.catch-and-release-forgotten/.stdout

```diff
@@ -2,4 +2,4 @@
 netcontrol debug (Debug-All): init
 netcontrol debug (Debug-All): add_rule: [ty=NetControl::DROP, target=NetControl::FORWARD, entity=[ty=NetControl::ADDRESS, conn=<uninitialized>, flow=<uninitialized>, ip=10.0.0.1/32, mac=<uninitialized>], expire=1.0 sec, priority=0, location=, out_port=<uninitialized>, mod=<uninitialized>, id=2, cid=2, _plugin_ids={\x0a\x0a}, _active_plugin_ids={\x0a\x0a}, _no_expire_plugins={\x0a\x0a}, _added=F]
 netcontrol debug (Debug-All): remove_rule (): [ty=NetControl::DROP, target=NetControl::FORWARD, entity=[ty=NetControl::ADDRESS, conn=<uninitialized>, flow=<uninitialized>, ip=10.0.0.1/32, mac=<uninitialized>], expire=1.0 sec, priority=0, location=, out_port=<uninitialized>, mod=<uninitialized>, id=2, cid=2, _plugin_ids={\x0a\x091\x0a}, _active_plugin_ids={\x0a\x091\x0a}, _no_expire_plugins={\x0a\x0a}, _added=T]
-Forgotten: , 10.0.0.1, [block_until=XXXXXXXXXX.XXXXXX, watch_until=XXXXXXXXXX.XXXXXX, num_reblocked=0, current_interval=0, current_block_id=2, location=<uninitialized>]
+Forgotten: , 10.0.0.1, [block_until=XXXXXXXXXX.XXXXXX, watch_until=1254722769.49206, num_reblocked=0, current_interval=0, current_block_id=2, location=<uninitialized>]
```

Comment:
Does this one need to be canonified at all?

Reply:
No, it does not. This is one of the mentioned cases where we could forego canonification completely - however that seemed like a bit of a separate issue.